docs/source/usecases/VirtualFunctions.md

```diff
@@ -111,6 +111,60 @@ deviceInstance->setAField(someHostValue); // set some field on the host
 
 This is the solution that the code teams we have talked to have said is the most productive way to solve the problem.
 
+## But what if I do not really need the V-Tables on the device side?
+
+There are cases where a lot of the logic of a code is implemented using virtual functions. Nevertheless, the performance critical part of the code might not use parent pointers and dynamic polymorphism to compute a result from data. In these cases, the device might not need to have a working virtual function table. Consider the following example:
+```c++
+struct Interface
+{
+    KOKKOS_DEFAULTED_FUNCTION
+    virtual ~Interface() = default;
+
+    KOKKOS_FUNCTION
+    virtual void operator()( const size_t, const Kokkos::View<int*> &) const = 0;
```

Comment:
You don't need the view to make your point.

Reply:
removed in [749fe2](https://github.com/kokkos/kokkos-core-wiki/pull/162/commits/749fe236c85d4ee7f9ed9417ccf160a081a17c40)